packages/teams-js/test/public/sharing.spec.ts

```diff
@@ -1,10 +1,12 @@
 import { app } from '../../src/public/app';
 import { ErrorCode } from '../../src/public/interfaces';
+import { FrameContexts } from '../../src/public/constants';
 import { sharing } from '../../src/public/sharing';
 import { Utils } from '../utils';
 
 describe('sharing_v1', () => {
   const utils = new Utils();
+  const allowedContext = ['content', 'sidePanel', 'task', 'stage', 'meetingStage'];
```

Comment:
Why is it that only these five out of the eight FrameContexts are allowed in this test?

Reply:
In sharing uses the above context and rest 3 are not allowed.
_The below code is from the sharing.ts_
 ` ensureInitialized(
      FrameContexts.content,
      FrameContexts.sidePanel,
      FrameContexts.task,
      FrameContexts.stage,
      FrameContexts.meetingStage,
    );`
    
   Also, unit test cases messages are misleading so to avoid the confusion I have limited the test cases for the allowed context.